rules/retrieve.smk

```diff
@@ -106,6 +94,234 @@ if (
             copy2(input["csv"], output["csv"])
 
 
+if (NUTS3_POPULATION_DATASET := dataset_version("nuts3_population"))["source"] in [
+    "primary",
+    "archive",
+]:
+
+    rule retrieve_nuts3_population:
+        input:
+            storage(
+                NUTS3_POPULATION_DATASET["url"],
+            ),
+        output:
+            f"{NUTS3_POPULATION_DATASET["folder"]}/nama_10r_3popgdp.tsv.gz",
+        retries: 2
+        run:
+            move(input[0], output[0])
+
+
+if (CORINE_DATASET := dataset_version("corine"))["source"] in ["archive"]:
+
+    rule retrieve_corine:
+        params:
+            url=CORINE_DATASET["url"],
+        output:
+            zip=f"{CORINE_DATASET["folder"]}/corine.zip",
+            directory=directory(f"{CORINE_DATASET["folder"]}"),
+            tif_file=f"{CORINE_DATASET["folder"]}/corine/g250_clc06_V18_5.tif",
+        run:
+            response = requests.get(params["url"])
+            with open(output.zip, "wb") as f:
+                f.write(response.content)
+
+            output_folder = Path(output["zip"]).parent
+            unpack_archive(output.zip, output_folder)
+
+
+elif (CORINE_DATASET := dataset_version("corine"))["source"] in ["primary"]:
+
+    rule retrieve_corine:
+        params:
+            apikey=config["secrets"]["corine"],
+        output:
+            zip=f"{CORINE_DATASET["folder"]}/corine.zip",
+            tif_file_path=f"{CORINE_DATASET["folder"]}/Results/u2018_clc2012_v2020_20u1_raster100m/DATA/U2018_CLC2012_V2020_20u1.tif",
```

Comment:
These rules yield files that are differently named. Is the content of the files identical (or similar)?

If yes, then the files should be named the same. That way it avoids confusion and it allows us to `build` a new version and directly upload it to the `archive.